content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -149,7 +149,7 @@ different Kubernetes components.
 | `ServiceTopology` | `false` | Alpha | 1.17 | |
 | `SetHostnameAsFQDN` | `false` | Alpha | 1.19 | |
 | `StartupProbe` | `false` | Alpha | 1.16 | 1.17 |
-| `StartupProbe` | `true` | Beta | 1.18 | |
+| `StartupProbe` | `true` | Beta | 1.18 | 1.19 |
```

Comment:
This is the wrong change I'm afraid.
The change you've made is right but, rather than replacing the line in place, keep it but move that amended line to the section https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features
then _add_ a new line showing that it goes GA in v1.20

Reply:
@sftim done